### PR TITLE
Show cleanup

### PR DESCRIPTION
**Pull request by arpariseau**
Clean-up on the show page, getting rid of ID information unnecessary to the user and just generally making it look nicer. Fix for the flag on the nav bar, as well as adding the district information to the representative index page.

### CODE REVIEW

**Review by Rostammahabadi — APPROVED** [review]

File: app/views/representatives/index.html.erb [hunk 1]
```diff
@@ -31,6 +31,11 @@
             <div class="card-body">
               <h6 class="card-title name"><%= member.full_name %>
               </h6>
+              <p class='card-district'><% if member.at_large %>
+              At-Large District
+              <% else %>
+              <%= member.district.ordinalize %> District
+              <% end %></p>
```
Comment (AntonioJacksonII) [inline app/views/representatives/index.html.erb:38]: Thank you for adding this!

**Review by AntonioJacksonII — APPROVED** [review]

File: app/views/house_members/show.html.erb [hunk 2]
```diff
@@ -27,29 +27,33 @@
       <!-- /.col-lg-8 -->
       <div class="col-lg-5">
         <h1 class='rep-name'><%= @rep.first_name %> <%= @rep.last_name %></h1>
+        <h4 class='rep-district'><%= "Representative from:" %><br/>
+          <%= ::States[@rep.state] + "'s " %>
+          <% if @rep.at_large %>At-Large District
+          <% else %><%= @rep.district.ordinalize + " District" %>
+          <% end %> </h4>
```
Comment (dborski) [inline app/views/house_members/show.html.erb:34]: Nice!

**Review by dborski — APPROVED** [review]

**Comment by dborski** [comment]
Nice job on this cleanup!

File: app/views/representatives/index.html.erb (hunk 1 above)
Comment (dborski) [reply app/views/representatives/index.html.erb:38]: Yes, thanks

**Pull request merged**